classes/util.php

```diff
@@ -48,5 +61,42 @@ public static function safe_normalize(string $string): string {
 
         return $normalised;
     }
-}
 
+    /**
+     * Remove accent character in text. Eg: Français -> Francais.
+     *
+     * @param string $string The input string.
+     * @return string The normal string without accents.
+     */
+    public static function remove_accent(string $string): string {
+        return preg_replace('/\p{Mn}/u', '', self::safe_normalize($string, Normalizer::FORM_KD));
```

Comment:
Is it true that every Mn = Mark, nonspacing unicode character is an accent?

Reply:
This function will remove all accents, not only for Europe but also other regions like Asian accents,... and even [TONE MARKS](https://vietnamesetypography.com/tone-marks/). Should I adjust this function to remove only Europe accent character?